src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Circle {
```

Comment:
The class 'Circle' is empty. According to the checklist, you should implement some behavior for this class. Typically, a Circle class would have properties like radius, and methods to calculate area and perimeter.